frontend/src/container/QueryBuilder/filters/QueryBuilderSearch/index.tsx

```diff
@@ -1,22 +1,21 @@
-import { Select, Spin, Tag, Tooltip } from 'antd';
+import { Select, Spin } from 'antd';
 import { useAutoComplete } from 'hooks/queryBuilder/useAutoComplete';
-import React, { useEffect, useMemo } from 'react';
+import React, { useCallback, useEffect, useMemo } from 'react';
```

Comment:
From `v17` onwards, you can skip importing `React`

`import { useCallback, useEffect, useMemo } from 'react';` will also work

Reply:
yeah we need to update eslint for this one